db/seeds.rb

```diff
@@ -5,3 +5,15 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+require 'nokogiri'
+require 'open-uri'
+
+doc = Nokogiri::HTML(open('http://www.learnathome.ru/blog/100-beautiful-words'))
+
+doc.search('//table/tbody/tr').each do |row|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.